freeipa/src/main/resources/freeipa-salt/salt/freeipa/common-install.sls

```diff
@@ -0,0 +1,32 @@
+/opt/salt/scripts/fix_next_update_grace_period.sh:
```

Comment:
instead of this script, could we use a salt state `file.replace`, as I checked the script it replaces a file content

Reply:
Thanks that works much better. I didn't know about `file.replace`.